browser/js/views/lecture/lectureModal.html

```diff
@@ -3,7 +3,17 @@
     <h1>Create Lecture</h1>
   </div>
   <div class="modal-body">
-    <input class="mainQuestion form-control" type="text" ng-model='lectureName' ng-keyup="$event.keyCode == 13 ? submitLecture() : null" placeholder='Lecture Title'>
+    <input class="mainQuestion form-control" type="text" ng-model='lectureName' placeholder='Lecture Title'>
+    <select id='lectureEmail' ng-model='lectureEmail'>
```

Comment:
Now not everybody would agree with me on this one, but I tend to avoid using `id` unless I think it's truly relevant (something global that there will only ever be one of). I think it tends to inspire less modular S/CSS which focuses on styling "this particular element right now" instead of "elements like this". For example, I notice the styling you're applying is `width: 100%`. Maybe just do something like `class="full-width"` instead (and then find or define some that class).